lib/winston-graylog2.js

```diff
@@ -72,6 +75,8 @@ var Graylog2 = winston.transports.Graylog2 = function(options) {
 
   this.graylog2 = new graylog2.graylog(options.graylog);
 
+  this.staticMeta = options.staticMeta || {};
```

Comment:
can we move this on line 75 just to keep it tidier with the other options defaulting?